packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -335,18 +337,24 @@ export class MgtTodo extends MgtTasksBase {
       `
       : html``;
 
+    const taskDeleteTemplate = this.readOnly
+      ? html``
+      : html`
+        <fluent-button class="task-delete"
+          @click="${() => this.removeTask(task.id)}"
+          aria-label="${this.strings.deleteTaskLabel}">
+          ${getSvg(SvgIcon.Delete)}
+        </fluent-button>
```

Comment:
If we're stacking attributes then I find it more readable to have the > character on its own line. For me its easier to see the indent levels